docs/reports/reports.rst

```diff
@@ -39,17 +39,21 @@ The **Details** tab on a Report contains the same options across all Reports and
    
    * Message Queue
 
-4. :doc:`Emails</channels/emails>`
+4. Audit Log
```

Comment:
Should this have page of its own given the others do? Guessing we might have to think where it would sit - maybe under Configuration?